database-migration/src/test/java/uk/ac/ebi/eva/dbmigration/mongodb/ExtractStatisticsFromVariantTest.java

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright 2016 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.dbmigration.mongodb;
+
+import com.github.fakemongo.Fongo;
+import com.mongodb.BasicDBList;
+import com.mongodb.BasicDBObject;
+import com.mongodb.DBCollection;
+import com.mongodb.DBCursor;
+import com.mongodb.DBObject;
+import com.mongodb.util.JSON;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.data.mongodb.core.MongoTemplate;
+
+import static junit.framework.TestCase.assertNotNull;
+import static junit.framework.TestCase.assertNull;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test {@link ExtractStatisticsFromVariant}
+ */
+public class ExtractStatisticsFromVariantTest {
+    private ExtractStatisticsFromVariant extractStatisticsFromVariant;
+
+    private static final String VARIANT_COLLECTION_NAME = "variantsCollection";
+
+    private static final String STATISTICS_COLLECTION_NAME = "statisticsCollection";
+
+    @Before
+    public void setUp() throws Exception {
+        extractStatisticsFromVariant = new ExtractStatisticsFromVariant();
+        extractStatisticsFromVariant.setVariantsCollectionName(VARIANT_COLLECTION_NAME);
+        extractStatisticsFromVariant.setStatisticsCollectionName(STATISTICS_COLLECTION_NAME);
+    }
+
+    @Test
+    public void variantWithoutStatisticsShouldNotMigrate() {
+        MongoTemplate mongoTemplate = new MongoTemplate(new Fongo("testServer").getMongo(), "variantWithoutStatistics");
+        DBCollection variantsCollection = mongoTemplate.getCollection(VARIANT_COLLECTION_NAME);
+        DBCollection statisticsCollection = mongoTemplate.getCollection(STATISTICS_COLLECTION_NAME);
+
+        DBObject variantWithoutSt = (DBObject) JSON
+                .parse("{ '_id' : 'Chr10_10000010_C_T', 'chr' : 'Chr10', 'start' : 10000010, 'files' : [ { 'fid' : 'ERZ123186', 'sid' : 'PRJEB10964', 'attrs' : { 'QUAL' : '255.0', 'CNV' : '64', 'TA' : 'Intergenic'}, 'fm' : 'GT:GL:GP:GQ:DP:AAC:LP', 'samp' : { 'def' : '0/0', '-1/-1' : [ 0, 1, 9, 10, 11, 12, 15, 18 ], '1/1' : [ 43 ] } } ], 'ids' : [ ], 'type' : 'SNV', 'end' : 10000010, 'len' : 1, 'ref' : 'C', 'alt' : 'T', '_at' : { 'chunkIds' : [ 'Chr10_10000_1k', 'Chr10_1000_10k' ] }, 'hgvs' : [ { 'type' : 'genomic', 'name' : 'Chr10:g.10000010C>T' } ] }");
```

Comment:
Please extract this string to an external variable that is split across multiple lines, like we do in the pipeline for [GtfStaticTestData](https://github.com/EBIvariation/eva-pipeline/blob/develop/src/test/java/uk/ac/ebi/eva/test/data/GtfStaticTestData.java) or [VariantData](https://github.com/EBIvariation/eva-pipeline/blob/develop/src/test/java/uk/ac/ebi/eva/test/data/VariantData.java).

This applies to the strings in all test methods.